sdk/nginx/Azure.ResourceManager.Nginx/CHANGELOG.md

```diff
@@ -4,12 +4,12 @@
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
+- Enable mocking for extension methods, refer this [document](https://aka.ms/azsdk/net/mocking) for more details.
```

Comment:
no version or release date update?

Reply:
Also, this should be located in `Features added`

Reply:
There is another ngix pr working on the upgrade https://github.com/Azure/azure-sdk-for-net/pull/40201